Source/SIMPLib/Common/Observer.cpp

```diff
@@ -42,6 +42,9 @@
 // -----------------------------------------------------------------------------
 Observer::Observer() = default;
 
+ Observer::Observer(QObject* parent)
+ : QObject(parent){}
+
```

Comment:
I don't think we need this change anymore since we went with the new API in FilterPipeline.